content/case-studies/persana-case-study.md

```diff
@@ -0,0 +1,213 @@
+---
+date: 2024-07-26T12:00:00Z
```

Comment:
If we don't have any specific date then before pushing to dev please note to change the date to the deploying day date.

Reply:
Okay vignesh.